README.md

```diff
@@ -167,6 +178,10 @@ The Photoshop APIs currently support creating and editing of Embedded Smart Obje
 - In order to update an embedded smart object that is referenced by multiple layers you only need to update one of those layers, the effect will be reflected in all layers referencing the same smart object.
 
 Please see the api docs for more information.
+https://git.corp.adobe.com/pages/dice/pie-in-the-sky/#api-Photoshop-document_operations
+
+Please find a JSON example of replacing a Smart Object within a layer.
```

Comment:
@khound I would just make this a live link: 
"[Smart Object Example Code](https://github.com/AdobeDocs/photoshop-api-docs#example-6-swapping-the-image-in-a-smart-object-layer)